### PR TITLE
[Enhancement](group commit) Optimize WAL insufficient space error messages

### PR DESCRIPTION
**Pull request by Yukang-Lian**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by Yukang-Lian** [comment]
run buildall

**Comment by Yukang-Lian** [comment]
run buildall

**Comment by Yukang-Lian** [comment]
run buildall

File: be/src/http/action/stream_load.cpp [hunk 1]
```diff
@@ -726,10 +726,11 @@ Status StreamLoadAction::_handle_group_commit(HttpRequest* req,
...
                           << " Bytes) exceeds the WAL (Write-Ahead Log) limit ("
-                          << max_available_size
-                          << " Bytes). So we set this load to \"group commit\"=sync_mode\" "
-                             "automatically.";
-                return Status::Error<EXCEEDED_LIMIT>("Stream load size too large.");
+                          << max_available_size << " Bytes). ";
+                std::stringstream ss;
+                ss << "There is no space for group commit async WAL. WAL dir info: "
+                   << ExecEnv::GetInstance()->wal_mgr()->get_wal_dirs_info_string();
+                return Status::Error<EXCEEDED_LIMIT>(ss.str());
```
Comment (mymeiyi) [inline be/src/http/action/stream_load.cpp:733]: log and returned status should use the same info?

File: be/src/olap/wal/wal_dirs_info.cpp [hunk 2]
```diff
@@ -105,6 +107,12 @@ void WalDirInfo::update_wal_dir_pre_allocated(size_t increase_pre_allocated,
     set_pre_allocated(increase_pre_allocated, decrease_pre_allocated);
 }
 
+std::string WalDirInfo::get_wal_dir_info_string() {
+    return "[" + _wal_dir + ": limit " + std::to_string(_limit) + " Bytes, used " +
+           std::to_string(_used) + " Bytes, pre allocated " + std::to_string(_pre_allocated) +
+           " Bytes, available " + std::to_string(available()) + "Bytes.]\n";
```
Comment (mymeiyi) [inline be/src/olap/wal/wal_dirs_info.cpp:113]: '\n' should not add here?

**Comment by Yukang-Lian** [comment]
run buildall

**Review by dataroaring — APPROVED** [review]
LGTM

**Review by mymeiyi — APPROVED** [review]

**Pull request merged**